src/StackExchange.Exceptional.AspNetCore/Extensions.cs

```diff
@@ -174,6 +174,7 @@ NameValueCollection TryGetCollection(Func<HttpRequest, IEnumerable<KeyValuePair<
                 { "Request Method", request.Method },
                 { "Scheme", request.Scheme },
                 { "Url", error.Url },
+                { "REMOTE_ADDR", context.Connection.RemoteIpAddress.ToString() }
```

Comment:
We need not populate this here, it's a backup

Reply:
Can you explain this? Don't undertand your intend wit this.